src/interaction/orbitalnavigator.cpp

```diff
@@ -403,6 +459,66 @@ void OrbitalNavigator::updateCameraStateFromStates(double deltaTime) {
         _camera->rotationQuaternion()
     };
 
+    if (_flyTo || _overview) {
```

Comment:
I think this method is already very long and hard to get an overview of. Could this code be moved out to a separate method? Also there are some assumptions about planets here like `planetRadius` and the constant `4` etc.

Reply:
This logic has been updated to only handle a generic linearflight to a set distance from the anchor node. Distinction between overview (flyout) and flyto is now made on the frontend instead.